app/src/main/java/com/elavon/converge/model/mapper/ContactlessMapper.java

```diff
@@ -1,77 +0,0 @@
-package com.elavon.converge.model.mapper;
```

Comment:
don't need this anymore - CL is either EMV (if it's EMV CL) or MSR (if it's MSD)